File: app/src/main/java/ua/in/quireg/anothermovieapp/common/MovieTrailer.java
package ua.in.quireg.anothermovieapp.common;

public class MovieTrailer {
    public String _id;
    public String iso_639_1;
    public String iso_3166_1;
    public String key;
    public String name;
    public String site;
    public String size;
    public String type;

    public MovieTrailer(String _id, String iso_639_1, String iso_3166_1, String key, String name, String site, String size, String type) {
        this._id = _id;
        this.iso_639_1 = iso_639_1;
        this.iso_3166_1 = iso_3166_1;
        this.key = key;
        this.name = name;
        this.site = site;
        this.size = size;
        this.type = type;
    }
}

File: app/src/main/java/com/anothermovieapp/repository/MovieTrailer.kt
package com.anothermovieapp.repository

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(primaryKeys = ["_id", ""], tableName = "trailers")
data class MovieTrailer(@PrimaryKey @ColumnInfo(name = "_id") var _id: String,
                        @ColumnInfo(name = "iso_639_1") var iso_639_1: String,
                        @ColumnInfo(name = "iso_3166_1") var iso_3166_1: String,
                        @ColumnInfo(name = "key") var key: String,
                        @ColumnInfo(name = "name") var name: String,
                        @ColumnInfo(name = "site") var site: String,
                        @ColumnInfo(name = "size") var size: String,
                        @ColumnInfo(name = "type") var type: String)